Add more detail about CLI parameters that no longer supported. (#625)
awslabs/mxnet-model-server: docs/migration.md

See more detail: [Install MMS](install.md)

## Command line interface

> MMS 1.0 made some parameter changes in `mxnet-model-server` command line tool. Following old command line parameters won't work any more:

Reviewer: ```suggestion
MMS 1.0 made some parameter changes in `mxnet-model-server` command line tool. The following command line parameters from previous versions will no longer function:
```

* --service, See [Register model](management_api.md#register-a-model) for how to override service entry-point.
* --gen-api, See [API description](inference_api#api-description) for how to generate your swagger client code.

> * --port, See [Configure MMS listening port](configuration.md#configure-mms-listening-port) for how to configure port.

Reviewer: ```suggestion
* --port, See [Configure MMS listening port](configuration.md#configure-mms-listening-port) for how to configure MMS listening ports.
```

* --host, See [Configure MMS listening port](configuration.md#configure-mms-listening-port) for how to bind to specific network interface.

> * --gpu, See [Config properties](configuration.md#other-properties) for how to limit number of GPUs.

Reviewer: ```suggestion
* --gpu, See [Config properties](configuration.md#other-properties) for how to limit the number of GPUs.
```

> * --log-file, See [Logging](#logging) section for how to configure logging.

Reviewer: ```suggestion
* --log-file, See [Logging](#logging) for how to specify a log file.
```

> * --log-rotation-time, See [Logging](#logging) section for how to configure logging.

Reviewer: ```suggestion
* --log-rotation-time, See [Logging](#logging) for how to configure log rotation.
```

> * --log-level, See [Logging](#logging) section for how to configure logging.

Reviewer: ```suggestion
* --log-level, See [Logging](#logging) for how to configure log level.
```

> * --metrics-write-to, See [Metrics](#metrics) section for how to configure metrics.

Reviewer: ```suggestion
* --metrics-write-to, See [Metrics](#metrics) for how to configure metrics.
```

> Please see [Command Line Interface](server.md#command-line-interface) for new parameters.

Reviewer: ```suggestion
For further information on the parameters' updates, please see the [Command Line Interface](server.md#command-line-interface) section of the server documentation.
```
